apps/files/src/store/actions.js

```diff
@@ -145,6 +145,15 @@ export default {
       }, { root: true })
     }).finally(() => {
       context.commit('UPDATE_FOLDER_LOADING', false)
+      client.users.getUser(user.id).then(res => {
```

Comment:
do we need this extra call? afaik login shall return the same data ....

look here https://github.com/owncloud/phoenix/blob/b3ae5b66374ba40c6bf09767088cde9e703fd2e6/src/store/user.js#L40

Reply:
damn - the information is not in there ....

Reply:
Yes, I looked there also and had the same revelation unfortunately